.github/workflows/deploy.yml

```diff
@@ -8,12 +8,56 @@ on:
     branches: [ master ]
 
 jobs:
-  deploy:
-    name: Deploy Demo App
+  build_browser_demo:
+    name: Build Browser Demo
+    runs-on: ubuntu-latest
+    env:
+        AWS_DEFAULT_REGION: us-east-1
+        AWS_DEFAULT_OUTPUT: text
+        AWS_ACCESS_KEY_ID: ${{secrets.AWS_ACCESS_KEY}}
+        AWS_SECRET_ACCESS_KEY: ${{secrets.AWS_SECRET_ACCESS_KEY}}
+    steps:
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          aws-region: us-east-1
+      - name: Checkout Package
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Install SAM CLI
+        run: script/github-action-awscli-installation
+      - name: Build Browser Demo
+        run: script/build-browser-demo
```

Comment:
We can just add steps here? It is just two steps so fine to not to add into its own script.

Reply:
Agree

Reply:
made the change